linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_configure_remote_server/ansible/shared.yml

```diff
@@ -0,0 +1,18 @@
+# platform = Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol,multi_platform_rhel,multi_platform_wrlinux
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = low
+- (xccdf-var var_audispd_remote_server)
+
+- name: Make sure that a remote server is configured for Audispd
+  lineinfile:
+    {{% if product in ["rhel8", "fedora", "ol8", "rhv4"] %}}
+    path: "/etc/audit/audisp-remote.conf"
+    {{% else %}}
+    path: "/etc/audisp/audisp-remote.conf"
+    {{% endif %}}
```

Comment:
For better readability I'd carve out the Jinja `if` out of the Ansible task and define a Jinja variable.
Something similar to: https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/bootloader-grub2/grub2_uefi_password/oval/shared.xml#L1